packages/swap/src/event-handlers/swapScheduled.ts

```diff
@@ -58,25 +57,22 @@ export default async function swapScheduled({
   if (origin.__kind === 'DepositChannel') {
     const depositAddress = origin.depositAddress.address;
 
-    const channels = await prisma.swapDepositChannel.findMany({
+    const channel = await prisma.swapDepositChannel.findFirst({
       where: {
         srcAsset: sourceAsset,
         depositAddress,
         issuedBlock: { lte: block.height },
       },
+      orderBy: { id: 'desc' },
     });
-    if (channels.length === 0) {
+    if (!channel) {
       logger.info(
         `SwapScheduled: SwapDepositChannel not found for depositAddress ${depositAddress}`,
       );
       return;
     }
-    assert(
```

Comment:
we did not think about this case when removing the `expiryBlock` in https://github.com/chainflip-io/chainflip-sdk-monorepo/pull/227

Reply:
after removing the `expiryBlock`, we cannot ensure that we have a single active channel in the database anymore because we dont know if a channel is expired. for now, we just assume that the latest channel is active